[2024-09-10T14:20:03.976+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T14:20:04.224+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: correction_Dag_Valeu_NULL.correct_transactions_anomalies scheduled__2024-09-10T14:15:00+00:00 [queued]>
[2024-09-10T14:20:04.258+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: correction_Dag_Valeu_NULL.correct_transactions_anomalies scheduled__2024-09-10T14:15:00+00:00 [queued]>
[2024-09-10T14:20:04.259+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-09-10T14:20:04.306+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): correct_transactions_anomalies> on 2024-09-10 14:15:00+00:00
[2024-09-10T14:20:04.332+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'correction_Dag_Valeu_NULL', 'correct_transactions_anomalies', 'scheduled__2024-09-10T14:15:00+00:00', '--job-id', '2439', '--raw', '--subdir', 'DAGS_FOLDER/Correction_dag_Valeur_NULL.py', '--cfg-path', '/tmp/tmpm_4fpais']
[2024-09-10T14:20:04.341+0000] {standard_task_runner.py:105} INFO - Job 2439: Subtask correct_transactions_anomalies
[2024-09-10T14:20:04.386+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=8571) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T14:20:04.388+0000] {standard_task_runner.py:72} INFO - Started process 8593 to run task
[2024-09-10T14:20:05.160+0000] {task_command.py:467} INFO - Running <TaskInstance: correction_Dag_Valeu_NULL.correct_transactions_anomalies scheduled__2024-09-10T14:15:00+00:00 [running]> on host 5f241b76205c
[2024-09-10T14:20:06.277+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='correction_Dag_Valeu_NULL' AIRFLOW_CTX_TASK_ID='correct_transactions_anomalies' AIRFLOW_CTX_EXECUTION_DATE='2024-09-10T14:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-10T14:15:00+00:00'
[2024-09-10T14:20:06.279+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T14:20:06.882+0000] {logging_mixin.py:190} INFO - Correction des anomalies dans la table Transactions r√©ussie.
[2024-09-10T14:20:06.884+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-10T14:20:07.345+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T14:20:07.348+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=correction_Dag_Valeu_NULL, task_id=correct_transactions_anomalies, run_id=scheduled__2024-09-10T14:15:00+00:00, execution_date=20240910T141500, start_date=20240910T142004, end_date=20240910T142007
[2024-09-10T14:20:07.638+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T14:20:08.006+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T14:20:08.010+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
